dart/lib/src/isolate_error_integration.dart

```diff
@@ -52,6 +52,14 @@ Future<void> handleIsolateError(
     final dynamic throwable = error.first;
     final dynamic stackTrace = error.last;
 
+    options.logger(
+      SentryLevel.error,
+      'Uncaught isolate error',
+      logger: 'sentry.isolateError',
+      exception: throwable,
+      stackTrace: StackTrace.fromString(stackTrace),
```

Comment:
can this throw? if `stackTrace` is null or it cant be a `StackTrace`, or any other reason? otherwise we need to wrap up in a try catch, maybe this is an instance of `StackTrace` already

Reply:
Yea this needs to be checked for null but IIRC this is a String because it has to be sent across isolates.

Reply:
yep, I'd then check all the cases, if it's null, if it's a StackTrace already, and do the `StackTrace.fromString` last case, also worth checking if `fromString` does not throw with an invalid string which is not a stack trace.

Reply:
From the isolate error handler doc:
>   /// The second element is a `String` representation of an accompanying
  /// stack trace, or `null` if no stack trace was provided.
  /// To convert this back to a [StackTrace] object, use [StackTrace.fromString].